docs/api/item.md

```diff
@@ -28,6 +32,125 @@ import Basic from '@site/static/usage/v7/item/basic/index.md';
 <Basic />
 
 
+## Content Types
+
+While items in a list take many forms, they typically support 5 different content types. However, not all of these content types should be used together at the same time. The following guide shows the different content types as well as how to properly utilize them in an application.
+
+### Supporting Visuals
```

Comment:
"Supporting Visuals"? I made it up 😂 I couldn't find consistent terminology across iOS/MD wasn't icon-specific. Lmk if you have other ideas for terminology.

Reply:
Haha no it's fine I was just curious because I didn't see it on the iOS human interface guidelines or MD guidelines. I think the only other way we've referred to it is as "Media" but I am not partial to anything.

Actually I just went and looked again and found it somewhat mentioned here: https://m3.material.io/components/lists/guidelines#4e4a0120-bc4d-4011-abad-dcc9f0a8637d